pages/news/2020-08-25-new-in-2-3-hotcue-colors.html

```diff
@@ -0,0 +1,141 @@
+title: New in 2.3: Color your Hotcues
+author: Holzhaus
+date: 2020-08-25 12:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+Hotcues are a neat thing and one of the major benefits of digital DJing.
+On analogue vinyl decks, you had had to put stickers to mark the different sections of a song and physically pick up, move and drop the needle to move to it.
+With the move to digital DJ setups, a new way to navigate tracks emerged: Hotcues now allow you to "bookmark" different spots in a track and quickly move to these positions in the blink of an eye, even while track is playing on the PA.
+
+Mixxx has supported them for more than 10 years now:
+Initial support for numbered hotcues has been added in 2009.
+
+However, it can be challenging to remember the correct number in the heat of the moment.
+Accidently pressing the wrong hotcue pad is not something you or your audience want to hear.
+
+## Better Hotcue Labels
+
+In 2015, support for hotcue labels was added to mitigate this problem.
+These can be arbitrary text that is shown on the waveforms.
+However, these were only shown on the scrolling waveforms, so you could not see them unless the track was close to that position.
+Also, it was cumbersome to edit labels because this was buried in the track Properties dialog that was only accessible from the library.
+
+In Mixxx 2.3, we overhauled the way you can interact with hotcues using the mouse:
+It's now possible to see the labels on the overview waveforms.
+You can edit them directly from the waveform by right clicking.
+
+If you have long, descriptive labels on hotcues that are close together, Mixxx will elide the labels so they don't overlap.
+You can still see the full label by hovering your mouse over an elided label.
+Mixxx will hide any other labels this would be drawn over so the text is legible.
+
+![Hotcue labels on overview waveforms]({% static "/images/news/hotcue-colors-labels.png" %})
+
+## Color your Hotcues
+
+But Mixxx 2.3 has yet another next major improvement for hotcues in the pipeline:
+Hotcues can now have individual colors to make them visually distinguishable.
+
+How you color code your hotcues is up to you - one example would be to set red cues 16 beats before the drop or chorus, yellow 8 beats before it and color the start of vocals blue.
+
+![Hotcue buttons and color picker]({% static "/images/news/hotcue-colors-picker.png" %})
+
+## Pick the colors YOU like
+
+By default, you can pick a color from the Mixxx Hotcue Color palette.
+That palette contains 9 distinct colors that are easy to distinguish.
```

Comment:
distinct / distinguish is redundant
```suggestion
That palette contains 9 colors that are easy to distinguish.
```